src/reference/asciidoc/logging.adoc

```diff
@@ -203,6 +203,8 @@ Extracting caller data can be rather expensive because the log event has to crea
 Therefore, by default, caller data associated with an event is not extracted when the event added to the event queue.
 You can configure the appender to include caller data by setting the `includeCallerData` property to `true`.
 
+Starting with _version 2.0.0_, the Logback `AmqpAppender` supports https://logback.qos.ch/manual/encoders.html[Logback encoders] with the `encoder` option. The `encoder` and `layout` options are mutually exclusive.
```

Comment:
@Monofraps ,

Thank you very much for the contribution.
Would you mind though to add some test-case on the matter?

Reply:
We need an answer maximum tomorrow. We have to release RC1 then.
Otherwise we'll do everything ourselves.

Thank you for the contribution anyway!

Reply:
Added test case here - https://github.com/garyrussell/spring-amqp/commit/75a8b31e729cbf461da17b63a59232fe3d177f0f - also added support for `outputPatternAsHeader`.

Reply:
Correction; https://github.com/garyrussell/spring-amqp/commit/bb29a8c5b238aa9c2f626ff2318e522ce1e275b8

Reply:
I'm terribly sorry for not responding! Something came up... Thanks for polishing and merging!